.github/workflows/deploy-prod.yml

```
@@ -0,0 +1,63 @@
+              cd /home/ubuntu/app
+
+              # 기존 프로세스 종료
+              PID=$(pgrep -f "app.jar")
+              if [ -n "$PID" ]; then
+                echo "Killing existing process: $PID"
+                kill -9 $PID
+              fi
+
+              # 새로운 프로세스 실행
+              nohup java -jar app.jar --spring.profiles.active=$SPRING_PROFILES_ACTIVE > app.log 2>&1 &
+            '
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)